Cargo.toml

```diff
@@ -46,5 +46,7 @@ lto = "thin"
 # [patch."https://github.com/pop-os/libcosmic"]
 # libcosmic = { path = "../libcosmic" }
 # cosmic-config = { path = "../libcosmic/cosmic-config" }
-# [patch."https://github.com/Smithay/client-toolkit"]
+[patch."https://github.com/Smithay/client-toolkit"]
 sctk = { git = "https://github.com/smithay/client-toolkit//", package = "smithay-client-toolkit", rev = "e63ab5f" }
+[patch.crates-io]
+sctk = { package = "smithay-client-toolkit", git = "https://github.com/pop-os/client-toolkit", branch = "wayland-resize" }
```

Comment:
Looks like the function added in `wayland-resize` isn't actually needed (though it looks like something that could be PRed and merged into upstream sctk?), and overriding both of these to `e63ab5f` works? (Or the most recent commit could be used, but it doesn't matter much.)

Unless that breaks vendoring or something...

Reply:
Oh, that branch is sctk 0.16, not 0.18? And sctk-adwaita is using it? So I guess  this does avoid duplicating that dependency. Though it's awkward.

Reply:
Ya, it's a bit awkward. I'll try figuring out a better solution for it.

Reply:
https://github.com/pop-os/iced/pull/88 & https://github.com/pop-os/libcosmic/commits/master seem to fix it

Reply:
Yes it should be able to be removed now